secure_redis/settings.py

```diff
@@ -4,12 +4,19 @@
 from django.core.exceptions import ImproperlyConfigured
 
 
-def get_secure_cache_opts():
+def get_secure_cache_opts(require_options=False):
```

Comment:
You asked a hard question :)
My first thoughts were that if we have cache settings without `option` then we don't want to fail. Since its not using `secure redis` library. but if someone used secure RQ job, it must fail if there are no `OPTIONS` added.

But actually I'm wondering if this code will be executed if no one called used secure redis cache settings! - I will double check and let you know.